### ISSUE TITLE
Add language to issue on explore 

### ISSUE THREAD

**Issue by vczb** [worknenjoy/gitpay#887]
Description 

### Why
I like to know what projects use React stack, so i propose its be possible add tags or language to set additional info. 

![image](https://user-images.githubusercontent.com/51221635/160246636-f4724e0f-247d-41db-8d71-98a6be797898.png)

Maybe can auto-import from the git code language or topics

examples:

![image](https://user-images.githubusercontent.com/51221635/160246896-5b2947df-e189-40b1-a275-6fd2ba0a4d09.png)

![image](https://user-images.githubusercontent.com/51221635/160246675-821111fd-bb1e-45b2-8e0e-90a3a1a4f765.png)

### Level
Medium

### Current behavior

I need open project one by one to know if its in React 

### Expected behavior

Easy way to know what language and main stacks the project use

Nice to have: filter by tag/language/etc

### Useful links
 https://github.com/topics

### How to reproduce the user scenario
- Scenario 1
   - Open explore
   - Look all issues
   - Know the project language
   - Find if necessary

### What you will need to complete this issue
Backend + Frontend

**Comment by Dheeraj4103** [comment]
Hey @alexanmtz Can I work on this issue ?

**Comment by alexanmtz** [comment]
hey @Dheeraj4103 , thanks for your interest in this one, let me check with @RemyaManoharan, are you still working on this?

**Comment by alexanmtz** [comment]
Hey, @Dheeraj4103, you can work on this one. It's pretty challenging, but follow the same approach for the Labels feature we implemented.

Github has the languages for the repository that you should extract to provide the languages. So maybe it's something to be added to the project not the issue.

As we do with Labels, you must work from the backend to create a new table and filter the languages on the UI.

Let me know if you need help!

I will add a bounty for this one 👌

**Comment by Dheeraj4103** [comment]
Sure @alexanmtz I will work on it, yes it seems challenging but I will refer to Labels feature to do it.
But it might take some time for me to do it, as I have joined a new organization this week. 
Hope you are okay with it.

**Comment by alexanmtz** [comment]
Yes @Dheeraj4103, take your time and let me know if you need anything

**Comment by Dheeraj4103** [comment]
Hey @alexanmtz, can you help me to `understand the flow on Label component`. As it will help me to work on language feature.
As languages are already present in label.` If I understand the flow of Label, then I can filter out languages from label data.`

**Comment by alexanmtz** [comment]
Hey @Dheeraj4103, thanks for reaching out.

It would be best to use the label just as a reference, but it's an entirely different model called 'programming languages.' As a label, we get from Github API when fetching a task, and you need to do the same with Languages, which it's defined on the Github API.

The label sometimes includes languages because it's generic, but they are different, the languages from Github return a list of the programming languages used on that repo. So, the idea is to have a dropdown to select programming languages from this programing language, and for the label you don't have to do anything, just inspiration of how to do it :)

It's an end-to-end task where you need to create the language model, migrations, and consume it when fetching a task from Github and getting the languages from the repository using the API (I guess for labels we get labels from the issue, so it is a little bit different logic).

Then, when exploring tasks in our issues list, they will list in a dropdown and filter those languages.

Let me know if you need any extra clarifications.

**Comment by Dheeraj4103** [comment]
Hey @alexanmtz Sorry, for replying this late, actually I was busy with my work.
Actually In October I have figured it out that how I can use GitHub API to get languages. 
Could you please help me to create a new table and Migration so that I can add languages for rest of task.

**Comment by alexanmtz** [comment]
hello @Dheeraj4103 , sure, let's go through it.

- You need to create a migration `sequelize migration:create --name modelname`
- Create a model
- Run the migration `npm run migrate` and `npm run migrate-test`
- Use the model to add Programming languages to the table when syncing with Github
- Get the dropdown and filters based on these languages populated on the model.

As I mentioned before, you can always use the Label model as reference, because they will be very similar.

Let me know if you have any questions.

**Comment by Dheeraj4103** [comment]
Hey, @alexanmtz can we take a google meet this weekend, so that you can help me to understand the flow, when a new Issue is added, because this is my first time contributing to large code base.

It will be a great help if we could talk, because then I can clear my doubts in one session.

**Comment by alexanmtz** [comment]
> Hey, @alexanmtz can we take a google meet this weekend, so that you can help me to understand the flow, when a new Issue is added, because this is my first time contributing to large code base.
> 
> It will be a great help if we could talk, because then I can clear my doubts in one session.

Hey @Dheeraj4103 , sorry for the late response. That's a good idea 👌 

So please join our Slack or contact me on the issue
https://gitpay.me/#/profile/task/899/add-language-to-issue-on-explore

So I can reply to you and send a invite for a Google Meet, so we can arrange the proper time for us :)

**Comment by Dheeraj4103** [comment]
Hey @alexanmtz Sorry for replying this late, actually  I'm working with a company and recent weeks have been busy.
And `I also don't have my personal laptop with me`, I can't work on this repo on my office laptop. But `don't worry I will be available from Christmas`, `I will have my personal laptop. So after Christmas we can take a meet if you have time.`

**Comment by Dheeraj4103** [comment]
Hey, @alexanmtz Created a PR for this issue, Please review this PR

https://github.com/worknenjoy/gitpay/pull/1165

**Comment by alexanmtz** [comment]
Merged #1165
